app/account/models.py

```diff
@@ -0,0 +1,30 @@
+from main import db as model
+
+
+class Account(model.Model):
+    """
+    Model for table account
+    """
+
+    __tablename__ = 'account'
```

Comment:
this is by default your model name in snake case, so can omit

Reply:
Done